hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/security/OMRenewDelegationTokenRequest.java

```diff
@@ -56,13 +61,31 @@ public OMRenewDelegationTokenRequest(OMRequest omRequest) {
 
   @Override
   public OMRequest preExecute(OzoneManager ozoneManager) throws IOException {
+    // We need to populate user info in our request object.
+    OMRequest request = super.preExecute(ozoneManager);
+
     RenewDelegationTokenRequestProto renewDelegationTokenRequest =
-        getOmRequest().getRenewDelegationTokenRequest();
+        request.getRenewDelegationTokenRequest();
+
+    AuditLogger auditLogger = ozoneManager.getAuditLogger();
+    Map<String, String> auditMap = null;
 
-    // Call OM to renew token
-    long renewTime = ozoneManager.renewDelegationToken(
-        OMPBHelper.convertToDelegationToken(
-            renewDelegationTokenRequest.getToken()));
+    long renewTime;
+    try {
+      Token<OzoneTokenIdentifier> token = OMPBHelper.convertToDelegationToken(
+          renewDelegationTokenRequest.getToken());
+      auditMap = buildTokenAuditMap(token);
+
+      // Call OM to renew token
+      renewTime = ozoneManager.renewDelegationToken(
+          OMPBHelper.convertToDelegationToken(
+              renewDelegationTokenRequest.getToken()));
```

Comment:
Nit: no need  to convert the token again.

```suggestion
      renewTime = ozoneManager.renewDelegationToken(token);
```